ush/config_defaults.yaml

```diff
@@ -159,10 +169,52 @@ platform:
   # If this is not set or set to an empty string, it will be (re)set to a 
   # machine-dependent value.
   #
+  # PARTITION_GRAPHICS
+  # Partition to which graphics tasks are submitted
+  #
+  # QUEUE_GRAPHICS
+  # Queue for graphics tasks
+  #
+  # PARTITION_ANALYSIS
+  # Partion to which analysis tasks are submitted
+  #
+  # QUEUE_ANALYSIS:
+  # The queue or QOS to which the task that runs a analysis is submitted.  
+  # If this is not set or set to an empty string, it will be (re)set to a 
+  # machine-dependent value.
+  #
+  # PARTITION_PRDGEN:
+  # If using the slurm job scheduler (i.e. if SCHED is set to "slurm"), 
+  # the partition to which the task that remaps output grids is submitted.  If 
+  # this is not set or set to an empty string, it will be (re)set to a 
+  # machine-dependent value.  This is not used if SCHED is not set to 
+  # "slurm".
+  #
+  # QUEUE_PRDGEN:
+  # The queue or QOS to which the task that prodgen is submitted.  
+  # If this is not set or set to an empty string, it will be (re)set to a 
+  # machine-dependent value.
+  #
+  # PARTITION_POST:
+  # If using the slurm job scheduler (i.e. if SCHED is set to "slurm"), 
+  # the partition to which the task that upp is submitted.
+  #
+  # QUEUE_POST:
+  # The queue or QOS to which the task that upp is submitted.  
+  # If this is not set or set to an empty string, it will be (re)set to a 
+  # machine-dependent value.
+  #
+  # RESERVATION:
+  # The reservation for major tasks.  
+  #
+  # RESERVATION_POST:
+  # The reservation for post tasks.  
+  #
```

Comment:
But the `parm/wflow` files get the there settings from config files, infact, we aready have partitions/queues in them already, so I am just extending that for jobs RRFS adds
```
    PARTITION_DEFAULT: '{{ platform.get("PARTITION_DEFAULT") }}'
    PARTITION_FCST: '{{ platform.get("PARTITION_FCST") }}'
    PARTITION_HPSS: '{{ platform.get("PARTITION_HPSS") }}'
    QUEUE_DEFAULT: '{{ platform.get("QUEUE_DEFAULT") }}'
    QUEUE_FCST: '{{ platform.get("QUEUE_FCST") }}'
    QUEUE_HPSS: '{{ platform.get("QUEUE_HPSS") }}'
   ```

Reply:
I'm suggesting that we don't necessarily have to set them explicitly here in config defaults. Here, the default queue is the main queue on a platform. FCST is special because it might need bigger nodes with more memory, and HPSS has internet access -- essentially it's the front-end node on platforms where that makes sense. Because each of these three function in separate ways for the workflow, we can choose from those 3 which task runs under which queue in the task definition.

There is no need for a one-to-one correspondence between available config defaults variables and what values are set as entities.


So, in the definition of the post jobs, we can modify the entry to specifically be `debug` if we want to change it for a given user/platform.